### ISSUE TITLE
[Feature] "The Avenger" support

### ISSUE THREAD

**Issue by NightFright2k19** [luciusDXL/TheForceEngine#440]
I tried loading the bonus level (extras.gob) from the Nightdive remaster and failed - at first. Internally, the level is labeled "AVENGER", which doesn't fit with any of the predefined campaign map names.

My solution was to rename the four affected gob entries to "SECBASE" in WDFuse. Maybe TFE could do that internally?

**Comment by luciusDXL** [comment]
Yeah, that has been something on my radar in terms of supporting the Remaster.

**Comment by NightFright2k19** [comment]
Excellent! Keep going. And it's awesome the port can use highres assets from the Nightdive port. Only the cutscenes are missing, but I'm sure you'll find a way to support those as well, eventually.

**Comment by luciusDXL** [comment]
Eventually HD Briefing, PDA, and cutscenes will be supported. I just moved them to low priority for now to focus on the editor and Outlaws suspport.

**Comment by NightFright2k19** [comment]
Good. Outlaws support is still a thing, especially since a remaster for that one seems rather unlikely to happen.

Before I forget: I created a mission briefing for the new map. I dunno if it's of any use for you since it would have to be bundled with extras.gob, but might still be a nice thing to have.

[avenger_dfbrief.zip](https://github.com/user-attachments/files/16646582/avenger_dfbrief.zip)

**Comment by TP555** [comment]
Hi

@NightFright2k19 

I tried loading the bonus level (extras.gob) from the Nightdive remaster and failed
Thats because no Jedi.LVL File is included in the Extras.gob.

I have made a "new" Fix now with your Mission Briefing.
Just Unpack that in your TFE Folder, and copy your extras.gob in the Mods Folder , and run the avenger.bat

My solution was to rename the four affected gob entries to "SECBASE" in WDFuse. Maybe TFE could do that internally?
Mayby yes, but this was a bad idea, i must rename both "ANIM" "DELT" Secbase Files inside the DFBRIEF.LFD to Avenger and Edit the Briefing.LST, Jedi.LVL was sometimes complitcated and a long Night :D to find all Errors ^^ , otherwise
The Mission Briefing Skips to load Mission Loading Screeen.

Cheers!

**Comment by TP555** [comment]
[The_Avenger_Final_Fix.zip](https://github.com/user-attachments/files/16710564/The_Avenger_Final_Fix.zip)
